utilities/src/main/java/io/syndesis/qe/TestConfiguration.java

```diff
@@ -29,6 +29,9 @@
 	public static final String SYNDESIS_CREDENTIALS_FILE = "syndesis.config.credentials.file";
 	public static final String SYNDESIS_VERSIONS_FILE = "syndesis.config.versions.file";
 
+	public static final String SYNDESIS_TEMPLATE_URL = "syndesis.config.template.url";
+	public static final String SYNDESIS_TEMPLATE_SA = "syndesis.config.template.sa.";;
```

Comment:
cosmetic issue: ";;" and "syndesis.config.template.sa." - remove dot from the end